.travis.yml

```diff
@@ -37,7 +36,7 @@ _test: &_test
     # Update python version in environment
     - sed -i -E 's/(python=)(.*)/\1'$TRAVIS_PYTHON_VERSION'/' ./environment.yml
     # Create conda environment
-    - conda env create -n radis-env -f environment.yml
+    - conda env create -n py36 radis-env -f environment.yml
```

Comment:
default RADIS test environnment is now python 3.8, is there any specific reason you want to add py36?

Reply:
hey @erwanp , **sorry for the late reply**. I just try with another YAML file. 😀

Reply:
I think the name should remain "radis-env" here